src/__tests__/datepicker.test.tsx

```diff
@@ -28,6 +28,60 @@ describe('Basic datepicker', () => {
     expect(setup()).toBeTruthy();
   });
 
+  describe('not read only or date picker only', () => {
+    it('accepts input', async () => {
+      const { getByTestId } = setup();
+      const datePickerInput = getByTestId('datepicker-input')
```

Comment:
Can you move this `getByTestId` to inside the `setup` function so that it can be reused without copy/paste?

Reply:
:+1: